assets/strings/eng.json

```diff
@@ -30,7 +30,7 @@
     "Format_Not_Supported": "Format not supported",
     "Translate_Webpage": "Translate a webpage",
     "Supported_Formats": "Supported formats: .txt, .rtf, .odp, .ods, .odt (LibreOffice/OpenOffice), .xlsx, .pptx, .docx (Microsoft Office 2003 and newer). <br/><i>Word 97 .doc files and pdf's are not supported.</i>",
-    "Download_Browser_Warning": "Your browser might not handle downloads correctly. Either right-click and save the link with the correct filename or open this page in <a href=\"https://www.mozilla.org/firefox/\">Firefox</a>.",
+    "Download_Browser_Warning": "Your browser does not handle downloads correctly so downloaded file will save as \"Unknown\", you might want to rename it. Alternatively you can open this page in a browser that handles downloads properly, such as <a href=\"https://www.mozilla.org/firefox/\">Firefox</a>.",
```

Comment:
Hmmm... I actually think leaving it at the old state might be OK because having vague messaging is not that bad and prevents having to re-translate every time something specific changes.

Reply:
Not sure if after that change you still can right click to save in Safari. Would need someone with this browser installed to try.

Reply:
Ah, that would be unfortunate and necessitate a change. I can check. If the change is required. all the other translation strings need to be deleted.

Reply:
Hm, you're right. It seems possible that a hybrid approach would work (i.e. adding `href` and `download` anyway).

Reply:
Also, would this problem with Safari not also affect IE?

Reply:
Yeah, it's impossible to download it in IE using right-click but my IE workaround uses a different method (`msSaveBlob` function implemented only by Microsoft) than Safari and so it saves with correct file name. And the warning never shows in IE.

Back to Safari, I'll do what you said and revert the string changes.

Reply:
> Yeah, it's impossible to download it in IE using right-click but my IE workaround uses a different method (msSaveBlob function implemented only by Microsoft) than Safari and so it saves with correct file name.

I see. Do you think it would hurt to make it possible to download via right-click as well? Users might be accustomed to doing different things and if it doesn't cost us much in terms of LOC/maintainability, I'm fine doing it.

> And the warning never shows in IE.

Great.

> Back to Safari, I'll do what you said and revert the string changes.
 
Perfect.